lib/conn_grpc/pool.ex

```diff
@@ -21,6 +21,15 @@ defmodule ConnGRPC.Pool do
           channel: [address: "localhost:50051", opts: []]
       end
 
+  You can also use the `:otp_app` option to load configuration from your application's config:
+
+      defmodule DemoPool do
+        use ConnGRPC.Pool, otp_app: :my_app
+      end
+
+  This will load configuration from `Application.get_env(:my_app, DemoPool, [])` and merge it
+  with the options passed to `child_spec/1`.
```

Comment:
@yordis can you add an example here of how `runtime.exs` would look like? You can use the same example from your PR description. This would help beginners figure out how to set the config.

Reply:
done!